src/components/button.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Button(props) {
+  const { onClick, name } = props;
+  const isZero = name === '0' ? ' btn-zero' : '';
+  return (
+    <button
+      className={`btn${isZero}`}
+      type="button"
+      onClick={onClick}
+    >
+      {name}
+    </button>
+  );
+}
+
+// Specifies the default values for props:
+Button.defaultProps = {
+  name: null,
+  onClick: null,
+};
```

Comment:
Default `color` for button is orange   ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)).

`wide` is boolean, true means that button take 50% of space (used for `0` button)   ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)).

Reply:
Hah, I read that but I thought, but only 5 out of the 10 + buttons are orange... So why isn't grey the default.